### DISCUSSION TITLE
How to set a permanent Metadata while on Auto-DJ

### DISCUSSION THREAD

**Discussion by renatopinto89** [AzuraCast/AzuraCast#4973] [General]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

 v0.14.1 Stable

### What happened?

Hello everyone and thank you for this amazing project.
I need to find a way to set the Now Playing metadata to be "You´re Listening to X Radio", while the station is on Auto-DJ, and not showing the Artist - Track name of each song.
How can I achieve that?
Thank you!

**Comment by ItsMitchh** [comment]
Hi there, to my knowledge, we have no ways of setting metadata while during a DJ session unless the media files are just 'You're listening to XYZ'.

**Reply by renatopinto89** [reply]
Hi Mitch! Thank you so much for replying, maybe I phrased that wrong, I meant during Auto-DJ (LiquidSoap). I was wondering if there's some way to edit it's configuration to replace the original metadata from the file to a permanent one which would be sent to SHOUTcasts or IceCasts "Now Playing" field
